Calculator/Program.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using CalculatorLibrary;
+using Newtonsoft.Json;
+
+namespace Calculator
+{
+    class Program
+    {
+        static void Main(string[] args)
```

Comment:
This method is very long! It should be separated out into smaller pieces. (Check other comments for suggestions on how to do this). 

And in general the `Main` method should just call other methods instead of having all of the logic built inside of it.